discovery-provider/src/tasks/index_user_bank.py

```diff
@@ -238,103 +545,68 @@ def process_transfer_instruction(
         elif user_id_account[1] == receiver_account:
             receiver_user_id = user_id_account[0]
     if sender_user_id is None:
+        # This can happen if the transaction happens before the userbank was indexed - clients should take care to confirm the userbank is indexed first!
```

Comment:
How will a client know that a lazily created USDC userbank was successfully created?

Reply:
Good question: Is there not a way on the client to send and confirm user bank creation?

Reply:
I guess we actually don't need the DN to index it, we just need it to be confirmed on chain... Since chain confirmation will ensure that they're indexed in the correct sequence, it being the same program and all